services-custom/s3-transfer-manager/pom.xml

```diff
@@ -212,6 +212,17 @@
             <version>${commons-codec.verion}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.github.hakky54</groupId>
+            <artifactId>logcaptor</artifactId>
+            <version>2.8.0</version>
```

Comment:
Let's manage versions in one place https://github.com/aws/aws-sdk-java-v2/blob/master/pom.xml#L119

Reply:
updated